src/controllers/employeeDetails.controller.js

```diff
@@ -24,6 +24,39 @@ const {employeeValidation,getId,getListValidation,recoveryParamsValidation,updat
     }
   };
 
+  //create Employee details with condition
+  const addOne = async (req ,res ,next) => {
+    const reqData = { ...req.query };
+    try {
+    const { employee_id, 
+      department_id, 
+      first_name, 
+      last_name, 
+      address,
+      email_id, 
+      phone_no,}  = await employeeValidation.validateAsync(req.body);
+    const { page_size,page_no } = await getListValidation.validateAsync(reqData);
```

Comment:
Please make sure your .eslint is configured.
Connect team if you donot have any idea regarding eslint.